src/server.js

```diff
@@ -1,10 +1,15 @@
 require('dotenv').config();
+
 const express = require('express');
-require('../database/config');
+
+const { logger } = require('./assests/logger');
+
+// eslint-disable-next-line no-unused-vars
+const db = require('../database/config');
```

Comment:
@bright258 you didn't resolve conflicts well, const db should not be there, only require should be

Reply:
Okay. I have made the correction